### ISSUE TITLE
Make it faster/easier to mark a movie or episode as watched.

### ISSUE THREAD

**Issue by cyr** [boxeehacks/boxeehack#393]
Having a bunch (500+) movies, having to go through them and mark them as watched takes quite a while, as a full refresh of the list is done each time one movie is set as watched. Could this be made in-place, instead? Marking a single movie in the list as watched, without actually refreshing the list?

This is not the only place where these types of improvements could be made either, responsiveness would increase alot if more in-place actions were made instead of reloading whole objects and lists.

**Comment by boxeehacks** [comment]
Agreed. I think it would be nice to do this with a keyboard shortcut similar to XBMC
